packages/integration-tests/src/components/delegates-focus-click/test-invoke-focus-on-click/integration/child/child.js

```diff
@@ -0,0 +1,9 @@
+import { LightningElement } from 'lwc';
+
+export default class extends LightningElement {
+    static delegatesFocus = true;
+
+    handleFocus(event) {
+        event.target.focus();
```

Comment:
does the button (`event.target`) already have the focus before calling `.focus()`?

Reply:
Yeah

Reply:
what confuses me is that the button will get the focus even if we don't execute this line. but i also guess that executing this line without the fix, will make the test fail.

Reply:
Yeah, it doesn’t make sense but we should be able to handle that!